examples/platform/silabs/wfx_rsi.h

```diff
@@ -71,13 +72,10 @@ typedef struct WfxEvent_s
 
 typedef struct wfx_rsi_s
 {
-    // TODO: Change tp WfxEventType_e once the event queue is implemented
-    EventGroupHandle_t events;
-    TaskHandle_t drv_task;
-    TaskHandle_t wlan_task;
-    TaskHandle_t init_task;
+    osThreadId_t drv_thread;
```

Comment:
This has been pointed out before but why does the different task handle/threadID need to be kept in this struck. (which is available over multiple interface).

I feel the task id would be contained to its owner.

Reply:
removed the threads from the structures and added to their required file